apps/showcase/project.json

```diff
@@ -145,7 +145,8 @@
         "generate-theme",
         "generate-dark-theme",
         "generate-horizon-theme",
-        "copy-training-assets"
+        "copy-training-assets",
+        "prepare-training"
```

Comment:
Ideally these should be added to the targets of the `serve` task in watch mode
At minimum, we need to added them in the `dependsOn` of the `serve` task